ci/render_pipelines_and_trigger_job

```diff
@@ -69,6 +69,7 @@ cleanup_pipelineruns
 
 # for local dev rendering use latest remote commit and no local one.
 if [ ! -z "${CC_CONFIG_DIR:-}" ]; then
+  echo "Local dev enviroment, using remote commit"
```

Comment:
is this a debugging leftover?

Reply:
No intended. we use the same script from Concourse trigger as well as for local development. in a local commit you often have commits not being pushed yet. So use commit from origin in this case so that pipeline works. I found it useful to echo a warning in this case